RocketCast/PlayerView.swift

```diff
@@ -22,6 +22,9 @@ class PlayerView: UIView {
         viewDelegate?.pausePodcast()
     }
     
+    @IBAction func changeSpeed(sender: UIButton) {
```

Comment:
Wouldn't it be better to use a dropdown element instead of a UIButton like this https://github.com/igongora/UIDropDown/blob/master/UIDropDown/UIDropDown.swift instead?